src/PKSim.Core/Services/SimulationModelCreator.cs

```diff
@@ -116,9 +116,6 @@ private void updateAllParametersFor(Simulation simulation, Individual individual
          var globalMoleculeContainer = simulation.Model.Root
             .GetSingleChildByName<IContainer>(molecule.Name);
 
-         hideParameter(globalMoleculeContainer.Parameter(CoreConstants.Parameters.ONTOGENY_FACTOR_GI));
```

Comment:
Not sure why they were explicitly hidden. Definitely not required...We were too conservative at the beginning :)